ign_rviz_plugins/include/ignition/rviz/plugins/TFDisplay.hpp

```diff
@@ -79,25 +103,19 @@ class TFDisplay : public MessageDisplay<tf2_msgs::msg::TFMessage>
   ~TFDisplay();
 
   // Documentation inherited
-  void LoadConfig(const tinyxml2::XMLElement * /*_pluginElem*/);
+  void LoadConfig(const tinyxml2::XMLElement * /*_pluginElem*/) override;
```

Comment:
```suggestion
  void LoadConfig(const tinyxml2::XMLElement * _pluginElem) override;
```

Reply:
I am not using the param _pluginElem at the moment. The definition in cpp file is the same as the declaration her and this `/*_pluginElem*/` is the same across all the files. If I uncomment this param in both header and cpp files without using it, it will lead to warning stating that the param has not been used.